### PR TITLE
frontend: Fix FSR

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
# Description
Properly fixes https://github.com/bottlesdevs/Bottles/issues/2312.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [x] Locally

### CODE REVIEW

**Comment by mirkobrombin** [comment]
FSR sharpening strength != quality. Quality depend on the resolution of the game. 
Sharpening  does not improve quality

**Comment by TheEvilSkeleton** [comment]
Thanks!

**Comment by TheEvilSkeleton** [comment]
Seems like an FSR dialog would be better.

File: bottles/frontend/ui/details-preferences.blp [hunk 1]
```diff
@@ -100,17 +100,44 @@ template DetailsPreferences : .AdwPreferencesPage {
...
+          "flat",
+        ]
+      }
 
       Switch switch_fsr {
         valign: center;
       }
     }
 
+    // .AdwComboRow combo_fsr {
```
Comment (jannuary) [inline bottles/frontend/ui/details-preferences.blp:122]: Shouldn't be here.

File: bottles/frontend/ui/dialog-fsr.blp [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          "suggested-action",
+        ]
+      }
+    }
+
+    .AdwPreferencesPage {
+
+      .AdwPreferencesGroup {
+        .AdwComboRow combo_upscaling_resolution_mode {
+          title: "Upscaling Resolution Mode";
```
Comment (jannuary) [inline bottles/frontend/ui/dialog-fsr.blp:43]: What does this mean?
Sounds really awkward.

Comment (TheEvilSkeleton) [reply bottles/frontend/ui/dialog-fsr.blp:43]: You're right. It's essentially FSR modes; the higher the quality, the lower the performance. What are your thoughts on "Quality Mode"? That's how AMD calls it.

**Review by mirkobrombin — APPROVED** [review]

**Review by jannuary — APPROVED** [review]

**Pull request merged**